src/common/types/issue-filing-needs-setting-content.ts

```diff
@@ -1,9 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { IssueFilingDialogProps } from '../../DetailsView/components/issue-filing-dialog';
+import { IssueFilingDialog, IssueFilingDialogProps } from '../../DetailsView/components/issue-filing-dialog';
 import { ReactSFCWithDisplayName } from '../react/named-sfc';
 import { IssueFilingNeedsSettingsHelpTextProps } from './../components/issue-filing-needs-settings-help-text';
 
 export type IssueFilingNeedsSettingsContentProps = IssueFilingDialogProps & IssueFilingNeedsSettingsHelpTextProps;
 
-export type IssueFilingNeedsSettingsContentRenderer = ReactSFCWithDisplayName<IssueFilingNeedsSettingsContentProps>;
+export type IssueFilingNeedsSettingsContentRenderer =
+    | ReactSFCWithDisplayName<IssueFilingNeedsSettingsContentProps>
```

Comment:
is this an extra `|`?

Reply:
Not sure, it was added by prettier and can't be modified otherwise.